File: year2020/month09/day0904/ms_0904.cpp
#include <iostream>

using namespace std;

int getMax(int *p);

int main() {
	int data[3];

	for (int i = 0; i < 3; i++)
		scanf_s("%d", &data[i]);

	cout << getMax(data) << endl;

	return 0;
}

int getMax(int *p) {
	int max = *p;

	p++;
	for (int i = 1; i < 3; i++, p++)
		if (max < *p)
			max = *p;

	return max;
}

File: year2020/month09/day0904/ms_0904.cs
using System;

namespace algorithm_50
{
    class ms_0904
    {
        static int getMax(int[] data) 
        {
            int max = data[0];

            for(int i = 1; i < data.Length; i++)
                if(max < data[i])
                    max = data[i];

            return max;
        }

        static void Main(string[] args)
        {
            string[] input = Console.ReadLine().Split(' ');
            int[] data = new int[3];
            
            for(int i = 0; i < data.Length; i++)
                data[i] = Convert.ToInt32(input[i]);

            Console.Write(getMax(data));
        }
    }
}
